packages/test-framework/src/fixture/pagePreset/PaymentStepAsGuestEU.ts

```diff
@@ -0,0 +1,17 @@
+import { Page } from '@playwright/test';
+
+import { ApiRequestsSender } from './ApiRequestsSender';
+import { CheckoutPagePreset } from './CheckoutPagePreset';
+
+export class PaymentStepAsGuestEU implements CheckoutPagePreset {
+    async apply(page: Page): Promise<void> {
+        const api = new ApiRequestsSender(page);
+
+        await api.addPhysicalItemToCart();
+        await api.setCurrency('EUR');
+        await api.dispose('Checkout is using EUR now');
```

Comment:
This is a wrapper for playwright's `dispose` API.
https://playwright.dev/docs/api/class-apirequestcontext#api-request-context-dispose
It is intended to be run only once at the end of a preset.